### DISCUSSION TITLE
Room in HA doesn't match the room in Mi app

### DISCUSSION THREAD

**Discussion by kmanan** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#446] [Q&A]
Hi, having a weird issue that I don't know how to resolve. If I select a specific room in HA (using the Lovelace map extractor card), it goes to a very different spot than the room I want it to clean. For example: I select the living room from the HA card, it goes to a spot in the dining room. How do I "reset" so that the rooms in the HA card match the rooms in the Mi app?

**Comment by PiotrMachowski** [comment]
Which vacuum do you have? Have you used "Generate room config" functionality?

**Reply by kmanan** [reply]
S6 MaxV. I tried that and the map added a bunch of rooms that don’t exist 
![IMG_6295](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/assets/17659155/252f18b5-60ad-4b01-8aad-cdc9e4ebe4c9)
![IMG_6296](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/assets/17659155/3710c25f-6cce-40f9-b9a4-fe57c0c43b18)

**Reply by PiotrMachowski** [reply]
Ok, now you can just remove ones that do not exist and everything should be fine 👍

**Reply by kmanan** [reply]
That did the trick 🙏🏽
